### ISSUE TITLE
Rendered video doesn't play until you seek in Firefox

### ISSUE THREAD

**Issue by dhasenan** [antiboredom/ffmpeg-explorer#8]
Linux, Ubuntu, Firefox 116.0.3-canonical-002.

Render the video, click the play button: the play button turns into a pause button but the video does not play. The progress bar does not advance.

Then, use the progress bar to seek: the video seeks properly, and when you stop seeking, it plays from there.

Basically, you need to seek for the video to actually work, but then it works normally, and everything else works right.

**Comment by antiboredom** [comment]
Yes - I've noticed this as well - I need to do some more testing to figure out what's causing the issue. Thanks for posting this.
